docs/end-user/components/cloud-services/terraform/tencent-vpc.md

```diff
@@ -0,0 +1,38 @@
+---
+title:  Tencent-Vpc
+---
+
+## Description
+
+Terraform configuration for Tencent Cloud VPC
+
+## Specification
+
+
+### Properties
+
+ Name | Description | Type | Required | Default 
+ ------------ | ------------- | ------------- | ------------- | ------------- 
+ vpc_cidr | The cidr block used to launch a new vpc when 'vpc_id' is not specified. |  | false |  
```

Comment:
what types are vpc_cidr, vpc_is_multicast and vpc_dns_servers?

Reply:
Fixed, thanks.